recipes/phylogenize/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.91" %}
+{% set sha256 = "3d428440f7a8aed3be18bbe094d3afe94f2d5357daa715d13e963c665384f2ab" %}
+{% set name = "phylogenize" %}
+
+package:
+  name: phylogenize
+  version: "{{ version }}"
+
+source: 
+  url: https://github.com/pbradleylab/phylogenize/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  host:
+    - perl
```

Comment:
Do the dependencies need their version pinned?

Reply:
Ah likely yes, if nothing else but to make it more robust! Thanks.

Reply:
Packages pinned to specific version.